tests/install_tests.rs

```diff
@@ -163,6 +182,36 @@ async fn install_locked() {
         Platform::current(),
         "python==3.9.0"
     ));
+
+    // Verify that the folders are present in the target directory using a task.
+    #[cfg(not(target_os = "windows"))]
+    pixi.tasks()
+        .add("which_python".into(), None, FeatureName::Default)
+        .with_commands(["which python"])
+        .execute()
+        .unwrap();
+    #[cfg(target_os = "windows")]
+    pixi.tasks()
+        .add("which_python".into(), None, FeatureName::Default)
+        .with_commands(["where python"])
+        .execute()
+        .unwrap();
```

Comment:
The only difference here seems to be `which` vs `where`, I would recommend to only put that part in `#[cfg()]` to reduce the clutter added by it.